File: result/repair/transcoder/java2python/python_repair/N_TH_NUMBER_WHOSE_SUM_OF_DIGITS_IS_TEN.py
def f_gold ( n ) :
    count = 0
    for curr in range ( 1 , 1 ) :
        sum = 0
        x=curr 
        while x>0 :
            sum = sum + x % 10
            x=x/10
        if sum == 10 :
            count += 1
        if count == n :
            return curr

File: result/translate/starchat/java2cpp/c++/N_TH_NUMBER_WHOSE_SUM_OF_DIGITS_IS_TEN.cpp
int f_gold ( int n ) {
    int count = 0 ;
    for ( int curr = 1 ; curr <= 1000000 ; curr ++ ) {
        int sum = 0 ;
        for ( int x = curr ; x > 0 ; x = x / 10 ) sum = sum + x % 10 ;
        if ( sum == 10 ) count ++ ;
        if ( count == n ) return curr ;
    }
}